org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/XMLLanguageServer.java

```diff
@@ -224,12 +226,12 @@ public void exit(int exitCode) {
 	}
 
 	@Override
-	public TextDocumentService getTextDocumentService() {
+	public XMLTextDocumentService getTextDocumentService() {
 		return xmlTextDocumentService;
 	}
 
 	@Override
-	public WorkspaceService getWorkspaceService() {
+	public XMLWorkspaceService getWorkspaceService() {

```

Comment:
Why you change to XMLWorkspaceService?

Reply:
The extension code registers a command and therefore need to register a handler for this command. I have added `onCommand(...)` to `XMLWorkspaceService` and now need to access the proper instance of the `XMLWorkspaceService`

Reply:
See https://github.com/eclipse/lemminx/pull/905#discussion_r501790802